modules/psap-node-feature-discovery-using-topology-updater.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_content-type: PROCEDURE
+[id="using-the-nfd-topology-updater_{context}"]
+= Using the NFD Topology Updater
+
+The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. The NFD Topology Updater communicates the information to nfd-master, which creates a `NodeResourceTopology` custom resource corresponding to all the nodes in the cluster. One instance of the NFD Topology Updater runs on each node of the cluster.
+
+.Procedure
+
+. To view available command line flags, run the `nfd-topology-updater -help` command. In a podman container:
```

Comment:
* Should be an unnumbered bullet for a single step
* The last sentence isn't a full sentence. Should it say "In a podman container, run the followingn command:"? Or just combine that into the first sentence?